doc/api/process.md

```diff
@@ -1503,6 +1503,10 @@ tarball.
   - `'Argon'` for the 4.x LTS line beginning with 4.2.0.
   - `'Boron'` for the 6.x LTS line beginning with 6.9.0.
   - `'Carbon'` for the 8.x LTS line beginning with 8.9.1.
+* `majorVersion` {number} The major version of Node.js.
+* `minorVersion` {number} The minor version of Node.js.
+* `patchVersion` {number} The patch version of Node.js.
+* `tag` {string} The semver tag for Node.js
```

Comment:
'semver tag` is not very descriptive. Perhaps a `prerelease` property with a simple bool would work here?

Reply:
maybe `prereleaseTag`? i want to expose the actual tag itself, not just if node is pre-release